src/main/webapp/level5.html

```diff
@@ -2,13 +2,17 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Level 5</title>
+    <title>Level 4</title>
```

Comment:
Are the names of the levels just not matching html now? Can this be switched for the file name to match the level number again?

Reply:
I thought I had switch this to level 5? I dont know why it says level 4 I will change that

Reply:
On my end it says level 5 for level5.html and level 4 for level4.html I think it looks weird because I switched the file names? I am not sure tbh